src/elements/selection-expansion-panel/selection-expansion-panel.ts

```diff
@@ -19,6 +19,10 @@ import '../divider.js';
 
 import style from './selection-expansion-panel.scss?lit&inline';
 
+const isSelectionPanelInput = (input: HTMLElement): boolean =>
+  !!input.closest('sbb-selection-expansion-panel') &&
+  !input.closest?.('sbb-selection-expansion-panel [slot="content"]');
+
```

Comment:
If the purpose of this function is just to distinguish the events coming from the unnamed slot from those coming from the `content` slot I think it's okay to remove this entire logic (since panel components are not allowed inside the `content` slot). And to foolproof it we can listen to the events on the unnamed slot anyway (instead of the host), so that any event coming from the content slot just won't trigger the listener. Don't you think? (hope this was clear)